Add back forms page (#5214)
postmanlabs/postman-docs: src/pages/docs/postman-flows/concepts/creating-a-form.md

The _Form Editor_ enables you to create and design your forms. It provides an intuitive interface for adding and configuring questions and managing user input.

In forms, _questions_ represent the input fields that users will interact with. Each question can have various question types. The following question types are supported:

* **Short answer** - Creates a single-line text box for users to provide concise responses.
* **Paragraph** - Creates a multiple-line text box for users to enter longer text or descriptions.
* **Checkboxes** - Creates a list of options with checkboxes for users to select multiple choices.
* **Multiple choice** - Creates a list of options with radio buttons for users to select a single choice.
* **Dropdown** - Creates a dropdown list with options that users can select from.
* **Required** - Enables you to make certain questions mandatory, ensuring that users must provide responses before submitting the form.

## Form workflow

To create a form, select a flow, then select **Form** from the bottom of the **Flows** sidebar.

<img alt="Create a form" src="https://assets.postman.com/postman-docs/v10/flows-create-form-v10.jpg" width="500px">

> You can name your form, add a description, and add questions by selecting the more actions icon <img alt="More actions icon" src="https://assets.postman.com/postman-docs/icon-more-actions-v9.jpg#icon" width="16px"> > **Edit** and using the available question types. Customize each question to gather specific data from users.

Reviewer: ```suggestion
Name your form, add a description, and add questions in the **Form Editor**, then select **Update & Publish**. To edit a published form, select the more actions icon <img alt="More actions icon" src="https://assets.postman.com/postman-docs/icon-more-actions-v9.jpg#icon" width="16px"> > **Edit**.
```

<img src="https://assets.postman.com/postman-docs/v10/flows-edit-form-v10.gif" alt="Edit a form" fetchpriority="low" loading="lazy" >

You can rename, duplicate, or delete questions as needed. You also have the flexibility to reorder questions to enhance the user experience.

> Once your form is ready, select **Preview** to view the form for testing. Select **Publish** to make the firm live.

Reviewer: ```suggestion
When your form is published, you can select **Preview** to view the form for testing.
```
